docs/resources/hostgroup.md

```diff
@@ -26,6 +26,8 @@ description: |-
 
 HostGroup resource
 
+~> **Note:** Exactly one of `host_ids` and `host_names` should be provided.
```

Comment:
Exactly one of `host_ids` and `host_names` is required?

Reply:
updated.